dotnet/aspnetcore/kubernetes/ArmTemplates/aks-template.json

```diff
@@ -71,12 +87,27 @@
                     "secret": "[parameters('servicePrincipalClientSecret')]"
                 }
             }
+        },
+        {
```

Comment:
I think you need to add a solution object as well

Reply:
Added